.cncf-maintainers

```diff
@@ -2,25 +2,19 @@ approvers:
 - OchiengEd
 - acornett21
 - anik120
-- everettraven
-- fabianvf
+- camilamacedo86
 - grokspawn
 - jberkhahn
 - joelanford
 - oceanc80
 - rashmigottipati
-- theishshah
-- varshaprasad96
 reviewers:
 - OchiengEd
 - acornett21
 - anik120
-- everettraven
-- fabianvf
+- camilamacedo86
```

Comment:
Hi @acornett21 

I moved myself form emirutus to approvers so that I can help more when needed.